### PR TITLE
fix: psp2 dependency in psp2kern

### PR DESCRIPTION
**Pull request by ZetaNumbers**

### CODE REVIEW

File: include/psp2kern/avcodec/jpegenc.h [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #ifndef _PSP2_AVCODEC_JPEGENC_H_
 #define _PSP2_AVCODEC_JPEGENC_H_
```
Comment (d3m3vilurr) [inline include/psp2kern/avcodec/jpegenc.h:7]: additionally, can you change this definition check to

```suggestion
#ifndef _PSP2_KERNEL_AVCODEC_JPEGENC_H_
#define _PSP2_KERNEL_AVCODEC_JPEGENC_H_
```

Comment (ZetaNumbers) [reply include/psp2kern/avcodec/jpegenc.h]: Oh, i think KERNEL prefix collides with psp2/kernel headers definition check prefix. Hm, should i make another PR or fix it here?

Comment (ZetaNumbers) [reply include/psp2kern/avcodec/jpegenc.h]: There are around ten of other cases like this too.

Comment (d3m3vilurr) [reply include/psp2kern/avcodec/jpegenc.h]: sure. you can make another pr :)

**Pull request merged**